### ISSUE TITLE
Incompleted [asrent] signs crashing server

### ISSUE THREAD

**Issue by SiuanSong** [NLthijs48/AreaShop#307]
#### Current behavior
Whenever I accidentally place incomplete [asrent] sign, my server times out.
Sign that crashes server:  (----'s denotes blank lines)
```yaml
[asrent]  
---------
---------
---------
```
Sign that doesn't crash server: (It recognizes as AS sign even if it's incorrect format)
```yaml
[asrent]
region
30 d
15
```

#### New behavior
Placing an incomplete sign doesn't crash server.

#### Steps to reproduce (for bugs)

1.  Place a sign with only [asrent] as first line and nothing else.
2. 
3. 
4. 

#### Your environment (for bugs)

* Server platform (Bukkit, Spigot, etc.): Spigot
* Server version: git-Spigot-3fb9445-2b6c9f4 1.11.2
* AreaShop version: 2.4.0
* WorldEdit version: 6.1.5;4651611
* WorldGuard version: 6.1.3-SNAPSHOT;c904242
* Vault version: 1.5.6-b49

Console Readout: https://pastebin.com/dNvh6zfk

#### Your config files (for bugs)

Cannot attach
Dropbox: https://www.dropbox.com/sh/2c5xfw4ztnrhhoj/AACajhuc_Tm1ApO7qMD3nsyma?dl=0

(This has happened to me about 5 times over the course of 2 months.)

**Comment by NLthijs48** [comment]
Could you send the actual log file from the `/logs` folder? The pastebin does not have line-endings and shows only a part of the error report.

There are some thing I can already comment on, first one being that placing a sign with only `[asrent]` on the first line is totally fine. In this case AreaShop will find the region you have placed the sign in and add that region to AreaShop. The price and duration are also optional, these will simply fallback to the setting specified in `default.yml`.

The code that handles the sign placement does not seem to do much special that can cause the server to stall, it simply asks WorldGuard which regions are at a certain location and chooses the region it needs from that.

There is a newer version of WorldGuard and WorldEdit available so I recommend at least updating those and see if you can cause the problem again.

**Comment by SiuanSong** [comment]
Had this happen again last night.  I have all current versions of WorldGuard, WorldEdit and AreaShop.

https://pastebin.com/UQjBZZhV

Less than 1 minute after IAmBlackBear created the shop region, I accidentally placed in incompleted [asrent] sign.  Server immediately timed out.  10 minutes later we ran /as del newshops to get rid of the shop region it automatically created and the server crashed.  After reboot all lighting in that entire region was gone for some bizarre reason.  hhttp://i.imgur.com/IYcvkbi.png
We were able to fix it using FAWE's //fix lighting
Thank you for your time, and sorry I didn't see your reply back to my initial report.
